projects/ADuCM3029_no_os_test_project/.cproject

```diff
@@ -1,398 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
these generates a lot of noise to remove a project and add it again. Why not just modify this one ?